CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#8030](https://github.com/rubocop-hq/rubocop/pull/8030): Refactoring of `Cop::Team` and `Cop::Commissioner`. ([@marcandre][])
```

Comment:
I think the users won't notice any changes from the refactoring, so maybe you can just skip the changelog entry.

Reply:
Ok